### PR TITLE
ENG-879: Implement pipeline for releasing dockers

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Test pipeline job available at rel.cd

### CODE REVIEW

File: docker/percona-docker.groovy [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        }
+        
+        stage('Save Image') {
+            when {
+                expression { params.DOCKER_EXPORT == true }
+            }
+            steps {
+                sh '''
+                    sudo bash -x docker_builder.sh --builddir=$(pwd)/tmp --export_container=1 --product=${DOCKER_NAME} --organization=${DOCKER_ORG} --version=${DOCKER_VERSION}
+                    sudo chmod -R 777 ${WORKSPACE}/tarball
```
Comment (EvgeniyPatlan) [inline docker/percona-docker.groovy:95]: why do we need such permissions?

Comment (Sudokamikaze) [reply docker/percona-docker.groovy:95]: As docker processes running as root, jenkins default user can't access/modify/read files without such permission

**Comment by Sudokamikaze** [comment]
> also please provide the link to the job

https://rel.cd.percona.com/job/Dockers-testingground/

File: docker/percona-docker.groovy (hunk 1 above)
Comment (EvgeniyPatlan) [reply docker/percona-docker.groovy:95]: you can do 
```
sudo usermod -aG docker jenkins
sh 'sg docker -c "...." '
```

Comment (Sudokamikaze) [reply docker/percona-docker.groovy]: Done

**Comment by Sudokamikaze** [comment]
In latest force push:
* Added gzip support for archiving images
* Reworked permissions
* Cleaned codebase, removed unneeded flags and reworked few as choice parameters instead of string
* Implemented tests

**Pull request merged**